.circleci/config.yml

```diff
@@ -421,3 +421,17 @@ workflows:
           <<: *build_on_tags
           requires:
             - build_x86_archlinux
+  test_nightly:
```

Comment:
Hm, why did that end up here?

Reply:
Ah, the PR base was wrong. Fixed.

Reply:
There are some commits from @erak , not sure how that happened

Reply:
probably the interactive rebase slipped that in? (maybe check `git log ...` for unexpected commits?)